### PR TITLE
#2027 - Offering form content changes

### PR DESCRIPTION
**Pull request by dheepak-aot**
### Bug Fix

For the bug mentioned in AC1 (Internal server error on offering submission when offering doesn't align with the delivery method of a program), I was not able to replicate the issue after trying all possible combinations of testing.

_Onsite Program and Online Offering_

Offering created in "Creation pending status"

_Online Program and Onsite Offering_

### Content Modification

image

image

image

image

### CODE REVIEW

**Review by andrewsignori-aot — APPROVED** [review]
Looks good and thanks for adding the scenarios and screenshots to the ticket also 👍

File: sources/packages/forms/src/form-definitions/educationprogramoffering.json [hunk 1]
```diff
@@ -170,8 +170,7 @@
       "showWordCount": false,
       "allowMultipleMasks": false,
       "addons": [],
-      "id": "ec8hh1",
-      "isNew": false
+      "id": "ec8hh1"
```
Comment (ann-aot) [inline sources/packages/forms/src/form-definitions/educationprogramoffering.json:173]: pls remove id

File: sources/packages/forms/src/form-definitions/educationprogramoffering.json [hunk 2]
```diff
@@ -2464,6 +2388,82 @@
...
+                  "label": "HTML",
+                  "attrs": [
+                    {
+                      "attr": "",
+                      "value": ""
+                    }
+                  ],
+                  "content": "Identify schedule breaks for the study period offering:\n<ul>\n  <li>\n    Study breaks of five or fewer consecutive calendar days that would normally be instructional days do not count towards study breaks.\n  </li>\n  <li>Study breaks cannot exceed three weeks, or 21 consecutive days. Excess break days will be deducted from overall funded weeks.</li>\n  <li>\n    Breaks in study may not exceed 10% (individually or cumulatively) of the study period offering length. Excess break days will be deducted from overall funded weeks.\n  </li>\n</ul>",
+                  "refreshOnChange": false,
+                  "customClass": "align-bullets mt-n2",
```
Comment (ann-aot) [inline sources/packages/forms/src/form-definitions/educationprogramoffering.json:2401]: for the info, why do we need `mt-n2`

**Review by ann-aot — APPROVED** [review]
LGTM 👍 Just a minor question

**Review by hellolynn-tbtb — APPROVED** [review]
Nice work @dheepak-aot !

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by sh16011993 — APPROVED** [review]
LGTM @dheepak-aot  👍

File: sources/packages/forms/src/form-definitions/educationprogramoffering.json (hunk 2 above)
Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/educationprogramoffering.json:2401]: The customClass is not part of the PR. It was only content change. Having said that, I believe we used mt-n2 reduce the vertical space between header and sub header. 

With mt-n2
image

Without mt-n2

image

Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/educationprogramoffering.json]: Thanks for the comment, I noticed something got flipped on the order and fixed it while investigating the comment.

**Pull request merged**